src/Uno.UI/UI/Xaml/UIElement.Pointers.Skia.cs

```diff
@@ -97,13 +107,39 @@ private void CoreWindow_PointerPressed(CoreWindow sender, PointerEventArgs args)
 
 				PropagateEvent(args, e =>
 				{
-					// Console.WriteLine($"PointerManager.Pressed [{e}/{e.GetHashCode():X8}");
+					if (this.Log().IsEnabled(LogLevel.Trace))
+					{
+						this.Log().Trace($"PointerManager.Pressed [{e}/{e.GetHashCode():X8}]");
+					}
+
 					var pointer = new Pointer(args.CurrentPoint.PointerId, PointerDeviceType.Mouse, false, isInRange: true);
-					var pointerArgs = new PointerRoutedEventArgs(args, pointer, e) { CanBubbleNatively = true };
-					TraverseAncestors(e, pointerArgs, e2 => e2.OnNativePointerDown(pointerArgs));
+					var pointerArgs = new PointerRoutedEventArgs(args, pointer, e);
+
+					Console.WriteLine($"Pointer pressed [{e}/{e.GetHashCode():X8}]");
```

Comment:
Codacy found an issue: [Remove this logging statement.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5924508)